_layouts/category.html

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+---
+<h2>Posts related to: {{ page.title }}</h2>
+
+{% unless page.content == '' %}
+  <p>{{ page.content }}</p>
+{% endunless %}
+
+{% for post in site.categories[page.category] %}
+<div class="post-preview">
```

Comment:
Der Preview hier weicht visuell noch von der Übersicht in /blog ab. Sollten wir das angleichen?